tests/test_arcaflow_plugin_horreum_client.py

```diff
@@ -70,18 +71,20 @@ def test_functional(self):
 
         responses.add(
             responses.POST,
-            input.horreum_keycloak_url
-            + "/realms/horreum/protocol/openid-connect/token",
+            f"{input.horreum_keycloak_url}"
+            "/realms/horreum/protocol/openid-connect/token",
```

Comment:
If you modify this module for some other reason, I would revert this particular change, but this isn't really a big deal.

The use of an f-string here doesn't improve the clarity of the code, and the line-break in the middle of what is otherwise a string literal makes it harder for the reader to discern what the parameters to the function are (i.e., it is easy to miss the fact that there is no trailing comma, so it looks like there are two positional parameters when there is in fact only one), while the `+` in the old code provides the prompt that the two lines are not separate.  This case is different from, e.g., the one at (new) line 83 where the content of the f-string includes stuff other than the variable to be formatted (although, in that case I would have put the `?` at the beginning of the next line, since it sort of corresponds to the subsequent lines' `&`'s 🙂).